js/tic_tac_toe.js

```diff
@@ -1,2 +1,84 @@
-// Your code here
-// Use the jQuery
+var count = 0;
+var blue_rows = [];
+var red_rows = [];
+var blue_columns = [];
+var red_columns = [];
+var $lFirst, $lSecond, $lThird, $rFirst, $rSecond, $rThird, $heading;
```

Comment:
Do keep in mind that these are all essentially global variables, since they are defined in the toplevel namespace. But that can be fixed, and is something we can work on.